<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emotionless</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_step">
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load_File"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Emulation">
    <property name="title">
     <string>&amp;Emulation</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="action_Stop"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Emulation"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Load_File">
   <property name="text">
    <string>Load File...</string>
   </property>
  </action>
  <action name="action_Load_Symbol_Map">
   <property name="text">
    <string>Load Symbol Map...</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About Emotionless</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>

